README.md

```diff
@@ -117,18 +120,19 @@ The package also includes gems for those using the [CDISC ADaM
 ADTTE](https://www.cdisc.org/standards/foundational/adam/adam-basic-data-structure-bds-time-event-tte-analyses-v1-0)
 data model.
 
-If columns `"PARAM"` or `"PARAMCD"` are present in the data frame, their
-values will be used to construct default labels in the `ggsurvfit()`
-figure.
+If columns `"PARAM"` or `"PARAMCD"` are present in the data frame passed
```

Comment:
... and ... are present based on your code. It is also requirement of CDISC that both are present.

Reply:
I mean that both PARAM and PARAMCD are required to be present in the ADTTE as per CDISC. You will only use PARAM.

Reply:
This is just a high-level overview from the README, and doesn't match 100% the requirements for the label to be used---and I am ok with that.